### PR TITLE
Video Dome documentation

### PR DESCRIPTION
**Pull request by DavidHGillen**
Here's the assorted documentation for the VideoDome

### CODE REVIEW

**Comment by deltakosh** [comment]
The PG link does not work for me: https://www.babylonjs-playground.com/#1E9JQ8#2
It generates a lot of warning on the console

**Comment by DavidHGillen** [comment]
I appear to be getting those errors now as well. Strange they were not there originally. What I'm relatively sure the issue is, comes from VideoTexture behaving unpredictably rather than an issue with the playground itself.

**Comment by deltakosh** [comment]
Well, it seems pretty reliable there: https://www.babylonjs-playground.com/#UU7RQ#61

**Comment by DavidHGillen** [comment]
Unreliable was probably the incorrect term, it is difficult to track down exactly what is erroring with it some times. I believe I have. but it will need a new nightly fix I believe as it turns out the video dome build on preview is missing a setting.

File: content/classes/3.1/VideoDome.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+PG_VERSION: 3.1
+TAGS:
+    - Node
+---
+## Description
+
+Display a 360 degree video on an approximately spherical surface, useful for VR applications or skyboxes.
+As a subclass of [Node](/classes/3.1/Node), this allow parenting to the camera or multiple videos with different locations in the scene.
+This class achieves its effect with a [VideoTexture](/classes/3.1/VideoTexture) and a correctly configured [BackgroundMaterial](/classes/3.1/BackgroundMaterial) on an inverted sphere.
+Potential additions to this helper includenon-infinite distance rendering effects for VR.
```
Comment (BabylonJSGuide) [inline content/classes/3.1/VideoDome.md]: please add a space between include and non.

**Comment by BabylonJSGuide** [comment]
The PG link  https://www.babylonjs-playground.com/#1E9JQ8#2 still not working will check again tomorrow after further nightly build

**Comment by deltakosh** [comment]
Should be good after merging a tiny fix

**Review by deltakosh — APPROVED** [review]

**Pull request merged**